server/src/server.js

```diff
@@ -17,7 +17,7 @@ app.use(express.json())
 app.use(cors());
 app.use('/', authRoutes);
 // Verify JWT middleware (all routes below require an access token)
-app.use(verifyJWT);
+// app.use(verifyJWT);
```

Comment:
I know this is is needed for testing purposes, but when pushing to master I think we should leave this in.